docs/src/content/docs/fr/guides/components.mdx

```diff
@@ -129,6 +129,50 @@ Ajoutez l'attribut `stagger` pour décaler verticalement la deuxième colonne de
 
 :::
 
+### Cartes (LinkCard)
```

Comment:
Ah, but this is a bit confusing isn’t it? Because the heading for the section above (for the `<Card />` component) is “Cards”, so now the table of contents is:

- Cards
- Cartes (LinkCard)

Which doesn’t seem right. We should translate both and translate “Link Cards” in a way that’s different from “Cards”. This would also match the translated headings for “Onglets” and “Icônes”.

Reply:
What if we translate Cards to Cartes and LinkCards to Cartes de Liaison?